zgrab_schema.py

```diff
@@ -202,7 +202,8 @@
     }),
     "session_ticket":SubRecord({
         "value":Binary(),
-        "length":Integer()
+        "length":Integer(),
```

Comment:
Length range according to RFC 5077 is 0 -> 2^16 - 1 but don't know what the real-world lengths are (hence the ZGrab patch). Definitely makes sense to be in bytes.